src/lib/routes/client-api/metrics.ts

```diff
@@ -34,8 +40,27 @@ export default class ClientMetricsController extends Controller {
 
         this.logger = getLogger('/api/client/metrics');
         this.clientInstanceService = clientInstanceService;
+        this.openApiService = openApiService;
         this.metricsV2 = clientMetricsServiceV2;
 
+        this.route({
+            method: 'post',
+            path: '',
+            acceptAnyContentType: true,
+            handler: this.registerMetrics,
+            permission: NONE,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['client'],
+                    operationId: 'registerMetrics',
+                    requestBody: createRequestSchema('clientApplicationSchema'),
```

Comment:
I think we need a separate schema for this endpoint: 

* The existing `clientApplicationSchema` was derived from Joi's `clientRegisterSchema`.
* This new schema should probably (also?) match Joi's `clientMetricsSchema`, with `bucket`, etc.

We should also add schema tests, and try out this new schemas manually through SDKs and the proxy.